app/components/tracker.ts

```diff
@@ -15,7 +15,8 @@ import { UserService }       from '../services/user';
 
 const HTML = require('../views/tracker.html');
 
-const MOBILE_WIDTH = 1100;
+const MOBILE_WIDTH          = 1100;
+const SHOW_SCROLL_THRESHOLD = 900;
```

Comment:
i think 900 is a bit much (but you're right, 50/100 was def too little). i think it should show up around when you've just hit the top of the first box - how's 400?